spec/spec_helper_acceptance.rb

```diff
@@ -35,6 +35,11 @@
 
     hosts.each do |host|
       on host, puppet('module', 'install', 'puppetlabs-stdlib'), acceptable_exit_codes: [0, 1]
+
+      if fact('osfamily') == 'Debian' && fact('operatingsystemmajrelease') == '8'
+        on host, 'echo deb http://ftp.debian.org/debian jessie-backports main >> /etc/apt/sources.list'
```

Comment:
We need to remove the one in the docker nodesets. Also, we need to do this for Debian 7 as well, maybe we can use the codename fact so we don't have to have two different conditionals?

Reply:
Basically, pull in these changes https://github.com/voxpupuli/modulesync_config/pull/403/files